features/facebookImport/src/context/importer-context.jsx

```diff
@@ -54,8 +54,16 @@ function updatePodNavigation(pod, history, handleBack, location) {
         : pod.polyNav.setActiveActions([]);
 }
 
-function updateTitle(pod) {
-    pod.polyNav.setTitle(i18n.t(`common:title`));
+function updateTitle(pod, location) {
+    pod.polyNav.setTitle(
+        i18n.t(
+            `navbarTitles:${
+                location.pathname === "/"
+                    ? "general"
```

Comment:
Maybe `index`? The logic bothers me a bit - depending on state we would either show `import` or `overview` initially - in which case we'd want to show the respective title, hm? `/` is more of a redirect (or should be), hm?

Suppose it's fine for now though, the titles are anyway largely the same.

Reply:
The thing is that a split second in the beginning location.pathname will be "/", which nobody will notice. This logic is just there to prevent it from breaking

Reply:
"index" instead of "general"?

Reply:
Yeah exactly (in the web world, `/` commonly has the server return files starting with `index`) -> I guess in that case I'd prefer an empty string! Wouldn't need to be in i18n at all, so we wouldn't have to worry about what to call it :P

Reply:
Done